src/crypto/store/indexeddb-crypto-store.js

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright 2017 Vector Creations Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import q from 'q';
+
+/**
+ * Internal module. indexeddb storage for e2e.
+ *
+ * @module
+ */
+
+const VERSION = 1;
+
+/**
+ * @implements {module:crypto/store/base~CryptoStore}
+ */
+export default class IndexedDBCryptoStore {
```

Comment:
This doesn't get displayed correctly when you `npm run gendoc`. The class is listed as `crypto/store/indexeddb-crypto-store` and it misses the constructor.

It's likely that this is happening because you're writing this in ES6 which JSDoc parses but does so incorrectly, for example members of class instances https://github.com/jsdoc3/jsdoc/issues/1301 - this is why we still use old-style ES5 in the JS SDK.